partner_prospect/models/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
@sudhir-erpharbor Add `Copyright`

Reply:
There were not any copyright in v11. What should it be exactly?

Reply:
In V8 Copyright here https://github.com/OCA/sale-workflow/blame/8.0/partner_prospect/models/res_partner.py#L2

But new version not found anywhere, keep as it is, And What Do you think about this senior?